TestRunner.psm1

```diff
@@ -115,13 +105,15 @@ function Write-Log {
     )
 
     $timestamp = Get-Date -Format "yyyy-MM-dd HH:mm:ss,fff"
-    $output = "[$timestamp] $Level $Message"
+    $output = "$timestamp - $Level - TestRunner - $Message"
```

Comment:
This is the format used in our Python code. I adjusted so that the powershell and Python log messages would have the same format. Thus you get output like this:

```
2022-04-28 12:25:42,689 - INFO - edfi_paging_test.performance_tester - Finished with paging volume test in 0.4940011501312256 seconds.
2022-04-28 12:25:42,798 - INFO - TestRunner - Test runner process complete
```